src/workspace.js

```diff
@@ -363,7 +363,6 @@ module.exports = class Workspace extends Model {
       })
     };
 
-    this.originalFontSize = null;
```

Comment:
Isn't this change breaking? It would be nice to have 
```js
this.originalFontSize = this.config.get('editor.defaultFontSize')
```

Reply:
Good point @aminya